api/envoy/api/v2/listener/listener.proto

```diff
@@ -216,4 +227,9 @@ message ListenerFilter {
 
     google.protobuf.Any typed_config = 3;
   }
+
+  // Decide when to enable/disable this listener filter on incoming traffic.
+  oneof switches {
+    ListenerFilterMatch disabled = 4;
```

Comment:
Per above comment the oneof here can be removed. Also, this needs a not-implemented-hide if you are not going to implement in this PR.

Reply:
add `not-implemented-hide` in both definition and usage